rest-api/rest-api-quiz.md

```diff
@@ -223,26 +223,19 @@
 - [x] common use cases
 - [ ] your tech stack
 
-#### Q34. What is the purpose of an OAuth refresh token?
+#### Q38. What is the purpose of an OAuth refresh token?
 - [ ] to share user profile information
 - [ ] to update an API configuration
 - [ ] to keep a web session active
 - [x] to retrive an access token
 
-#### Q35. What is Time to First Hello World?
+#### Q39. What is Time to First Hello World?
 - [x] how long it takes for a developer to do something with your API
 - [ ] how long it takes to start a new programming language
 - [ ] how long it takes to install your SDK
 - [ ] how long it takes to read your documentation
 
-#### Q35. What is Time to First Hello World?
-- [x] how long it takes for a developer to do something with your API
-- [ ] how long it takes to start a new programming language
-- [ ] how long it takes to install your SDK
-- [ ] how long it takes to read your documentation
-
-
-#### Q36. Which response header tells the client and intermediaries that the response is not to be cached anywhere?
```

Comment:
And this questions, should be present!

Reply:
But this questions with answers should stay? From you PR I see that you removed question
```
#### Q36. Which response header tells the client and intermediaries that the response is not to be cached anywhere?
- [ ] Cache-State: none
- [ ] Expires:-1
- [ ] Cache-Control: no-cache
- [x] Cache-Control: no-store
```